ittybittyapps/appstoreconnect-cli
# Expire a build

# 📝 Summary of Changes

Changes proposed in this pull request:

- expire a build

# ⚠️ Items of Note
- once a build has been expired it can't be undone
- As per the discussion modifying the usesNonExemptEncryption will be done later

## 🔨 How To Test

- Do a read before executing an expire command
   ```swift run asc testflight builds read iba.test2 ```

![Screen Shot 2020-05-06 at 4 37 34 pm](https://user-images.githubusercontent.com/61398249/81144709-f75bd800-8fb7-11ea-8790-35c5298c5829.png)

- Execute the expire command as follows
    ```swift run asc testflight builds expire iba.test2 1.0 6 true```

- Check the change by executing another read command
    ```swift run asc testflight builds read iba.test2 ```

![Screen Shot 2020-05-06 at 4 36 47 pm](https://user-images.githubusercontent.com/61398249/81144626-d3989200-8fb7-11ea-80c0-0ef149981b82.png)
